config/kubernetes/sig-contributor-experience/teams.yaml

```diff
@@ -47,18 +50,20 @@ teams:
   contributor-comms:
     description: alert members of new issues labeled with contributor comms
     maintainers:
+    - mfahlandt
+    - kaslin
+    - chris-short
```

Comment:
Please add yourself and others to members list. `maintainers` list is only for @kubernetes/owners.